code/modules/clothing/clothing.dm

```diff
@@ -931,3 +931,30 @@ BLIND     // can't see anything
 	slot_flags = SLOT_GLOVES
 	gender = NEUTER
 	var/undergloves = 1
+
+// Tape
+
+/obj/item/clothing/glasses/tape
+	name = "length of tape"
+	desc = "A length of sturdy tape."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "tape_cross"
+	item_state = "tape_cross"
+	body_parts_covered = HEAD
+	slot_flags = SLOT_MASK | SLOT_EYES
+	body_parts_covered = FACE|EYES
+
+	var/voicechange = 1
+	var/say_messages = list("Mmfph!", "Mmmf mrrfff!", "Mmmf mnnf!")
+	var/say_verbs = list("mumbles", "says")
+
+/obj/item/clothing/glasses/tape/equipped(mob/user, slot)
+	if (slot == slot_glasses)
+		tint = TINT_BLIND
+	else
+		tint = TINT_NONE
+
+	. = ..()
+
+/proc/is_muzzle(thing)

```

Comment:
Duplicate. There's a criminally underused proc called `/mob/living/is_muzzled()`, defined in `say.dm`. Do this instead, within that same file:

```dm
/mob/living/carbon/human/is_muzzled()
	if (..())
		return TRUE

	if (wear_mask)
		if (istype(wear_mask, /obj/item/clothing/mask/muzzle) || istype(wear_mask, /obj/item/clothing/glasses/tape))
			return TRUE

	return FALSE
```

Reply:
Or:
```dm
/mob/living/carbon/human/is_muzzled()
    return ..() || istype(wear_mask, /obj/item/clothing/mask/muzzle) || istype(wear_mask, /obj/item/clothing/glasses/tape)
```